components/Map.js

```diff
@@ -195,15 +209,50 @@ class Map extends React.Component {
     layer.bindPopup(popupContent);
   };
 
+  // Make mode selection nicer for displaying in FeatureCounts
+  formatModeSelection = modeSelection => {
+    if (modeSelection == 'all') {
+      return 'All';
+    } else if (modeSelection == 'bike') {
+      return 'Bike';
+    } else if (modeSelection == 'mv') {
+      return 'Motor vehicle';
+    } else {
+      return 'Pedestrian';
+    }
+  };
+
+  // Make dataset selection nicer for displaying in FeatureCounts
+  formatDataSelection = dataSelection => {
+    return dataSelection == 'crash' ? 'crashes' : 'fatalities';
+  };
+
+  // Make selected dates nicer for displaying in FeatureCounts
+  formatDate = date => {
+    return format(date, 'MM/D/YY');
+  };
+
   render() {
     return (
       <div>
         {/* make map take up entire viewport with room for the navbars */}
         <div
           style={{ height: 'calc(100vh - 125px)' }}
           ref={this.setMapEl}
-          lastUpdated={this.state.lastUpdatedDate}
-        />
+          lastupdated={this.state.lastUpdatedDate}
```

Comment:
What does the `lastupdated` attribute on a `<div>` do?

Reply:
nothing, that's a typo - thanks for finding